LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2019 Softonic International S.A.
+Copyright 2021 Softonic International S.A.
```

Comment:
2022 could be better :-)

Reply:
Haha ok, I just copied the template, but you're right.